pkg/controller/dnszone/awsactuator.go

```diff
@@ -51,14 +54,9 @@ func NewAWSActuator(
 	dnsZone *hivev1.DNSZone,
 	awsClientBuilder awsClientBuilderType,
 ) (*AWSActuator, error) {
-	// This allows for using host profiles for AWS auth.
-	var regionName string
-
-	if dnsZone != nil && dnsZone.Spec.AWS != nil {
-		regionName = dnsZone.Spec.AWS.Region
-	}
-
-	awsClient, err := awsClientBuilder(secret, regionName)
+	// Route53 is a regionless service, we specify a default region just for the purpose of creating
+	// our client configuration.
+	awsClient, err := awsClientBuilder(secret, defaultRegionEndpoint)
```

Comment:
Here be dragons. Is this what we expect if we remove the region?

Reply:
That is what the DNSEndpoint code does now. I don't know if this will work for special regions like gov-cloud, though.

Reply:
Ok good, we can cross that bridge if we come to it, probably via a HiveConfig option.